Test/PesterLogger/RemoteLogCollector.ps1

```diff
@@ -44,7 +44,7 @@ class FileLogSource : LogSource {
                 }
             } else {
                 foreach ($File in $Files) {
-                    $Content = Get-Content -Raw $File
+                    $Content = Get-Content -Force -Raw $File
```

Comment:
Since `Out-File -Append` does not do a read lock on a file, we can remove `-Force` flag.